horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/SchemaServiceTest.java

```diff
@@ -273,12 +273,84 @@ public void testFindUsages(TestInfo info) throws InterruptedException  {
 
       assertNotEquals(0, report.data.size());
 
-      List<SchemaService.LabelLocation> usages = jsonRequest().get("/api/schema/findUsages?label=".concat("category"))
-                .then().statusCode(200).extract().body().as(List.class);
+      List<SchemaService.LabelLocation> usages =
+              jsonRequest().get("/api/schema/findUsages?label=".concat("category"))
+              .then().statusCode(200).extract().body().as(List.class);
 
       assertNotNull(usages);
+   }
+
+   @org.junit.jupiter.api.Test
+   public void testCreateSchemaAfterRun() throws InterruptedException {
```

Comment:
Should we mention that the run is an array run in the test name? The current Horreum implementation handles arrays and json objects in a significantly different manner.
Maybe `testCreateSchemaAfterRunUsingArrayData`?